### DISCUSSION TITLE
I have a problem with tooltip combined with dialogs

### DISCUSSION THREAD

**Discussion by lo-dominici** [microsoft/fluentui#32423] [Q&A] [answered]
My purpose is to open a dialog using a button inside a tooltip content. To implement a simple demo I used the original code in stackblits and combined the 2 components. All is working except for one thing: when the dialog is closed using the close button or the 'x' button, the tooltip remains visible and is not hidden until hovered. Instead if I close the dialog clicking outside of it, the tooltip is hidden correctly. What am I missing? 
[Demo link](https://stackblitz.com/edit/7mt1j1?file=src%2Fexample.tsx).
Thanks for your help.

**Comment by ValentinaKozlova** [comment, marked as answer]
Hello @lo-dominici, have you tried to use controlled tooltip?
https://react.fluentui.dev/?path=/docs/components-tooltip--docs#controlled

**Reply by lo-dominici** [reply]
Hi, I did try it and the problem is now gone, thank you for the tip!
